From 2851e9a7614ac6a14f8922528740742662b738d3 Mon Sep 17 00:00:00 2001
From: Andrew Hsieh <andrewhsieh@google.com>
Date: Thu, 14 May 2015 14:34:02 +0800
Subject: [PATCH] Add --fix-cortex-a53-843419

Add but ignore this option which is passed from gcc4.9/aarch64
by default unless user specify -mno-fix-cortex-a53-843419.

Change-Id: I5256dd4f348809f7f34821c4ccebe702eada720b
---
 tools/mcld/lib/OutputFormatOptions.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/tools/mcld/lib/OutputFormatOptions.cpp b/tools/mcld/lib/OutputFormatOptions.cpp
index a77b4a8..0eccd9d 100644
--- a/tools/mcld/lib/OutputFormatOptions.cpp
+++ b/tools/mcld/lib/OutputFormatOptions.cpp
@@ -181,6 +181,12 @@ llvm::cl::opt<bool> ArgFixCortexA53Errata835769(
     llvm::cl::ZeroOrMore,
     llvm::cl::init(false));
 
+llvm::cl::opt<bool> ArgFixCortexA53Errata843419(
+    "fix-cortex-a53-843419",
+    llvm::cl::desc("Fix AArch64 A53 Errata 843419."),
+    llvm::cl::ZeroOrMore,
+    llvm::cl::init(false));
+
 // Not supported yet {
 llvm::cl::opt<bool> ArgExportDynamic(
     "export-dynamic",
-- 
2.2.0.rc0.207.ga3a616c

